modules/objectdb_profiler/editor/data_viewers/summary_view.cpp

```diff
@@ -36,71 +36,51 @@
-	content_wrapper->add_theme_style_override(SceneStringName(panel), content_wrapper_sbf);
-	content_wrapper->add_child(mc);
-	add_child(content_wrapper);
+	content_wrapper->set_theme_type_variation("PanelContainerDarkFlat");
+	content_wrapper->add_child(margin);
 
 	VBoxContainer *content = memnew(VBoxContainer);
-	mc->add_child(content);
-	content->set_anchors_preset(LayoutPreset::PRESET_FULL_RECT);
-
-	PanelContainer *pc = memnew(PanelContainer);
-	Ref<StyleBoxFlat> sbf;
-	sbf.instantiate();
-	sbf->set_bg_color(EditorNode::get_singleton()->get_editor_theme()->get_color("dark_color_3", "Editor"));
-	pc->add_theme_style_override("panel", sbf);
-	content->add_child(pc);
-	pc->set_anchors_preset(LayoutPreset::PRESET_TOP_WIDE);
+	margin->add_child(content);
+	content_wrapper->set_anchors_and_offsets_preset(PRESET_FULL_RECT);
+	add_child(content_wrapper);
+
+	PanelContainer *panel = memnew(PanelContainer);
+	panel->set_theme_type_variation("PanelContainerDarkFlat");
+	content->add_child(panel);
 	Label *title = memnew(Label(TTRC("ObjectDB Snapshot Summary")));
-	pc->add_child(title);
-	title->set_horizontal_alignment(HorizontalAlignment::HORIZONTAL_ALIGNMENT_CENTER);
-	title->set_vertical_alignment(VerticalAlignment::VERTICAL_ALIGNMENT_CENTER);
+	panel->add_child(title);
+	title->set_horizontal_alignment(HORIZONTAL_ALIGNMENT_CENTER);
+	title->set_vertical_alignment(VERTICAL_ALIGNMENT_CENTER);
 
 	explainer_text = memnew(CenterContainer);
-	explainer_text->set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
-	explainer_text->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
+	explainer_text->set_v_size_flags(SIZE_EXPAND_FILL);
 	content->add_child(explainer_text);
 	VBoxContainer *explainer_lines = memnew(VBoxContainer);
 	explainer_text->add_child(explainer_lines);
-	Label *l1 = memnew(Label(TTRC("Press 'Take ObjectDB Snapshot' to snapshot the ObjectDB.")));
-	Label *l2 = memnew(Label(TTRC("Memory in Godot is either owned natively by the engine or owned by the ObjectDB.")));
-	Label *l3 = memnew(Label(TTRC("ObjectDB Snapshots capture only memory owned by the ObjectDB.")));
-	l1->set_horizontal_alignment(HorizontalAlignment::HORIZONTAL_ALIGNMENT_CENTER);
-	l2->set_horizontal_alignment(HorizontalAlignment::HORIZONTAL_ALIGNMENT_CENTER);
-	l3->set_horizontal_alignment(HorizontalAlignment::HORIZONTAL_ALIGNMENT_CENTER);
-	explainer_lines->add_child(l1);
-	explainer_lines->add_child(l2);
-	explainer_lines->add_child(l3);
-
-	ScrollContainer *sc = memnew(ScrollContainer);
-	sc->set_anchors_preset(LayoutPreset::PRESET_FULL_RECT);
-	sc->set_v_size_flags(SizeFlags::SIZE_EXPAND_FILL);
-	sc->set_h_size_flags(SizeFlags::SIZE_EXPAND_FILL);
-	content->add_child(sc);
+	Label *label = memnew(Label());
+	label->set_text(TTRC("Press 'Take ObjectDB Snapshot' to snapshot the ObjectDB.\n\nMemory in Godot is either owned natively by the engine or owned by the ObjectDB.\n\nObjectDB Snapshots capture only memory owned by the ObjectDB."));
```

Comment:
The double newlines look excessive

<img width="601" height="190" alt="Image" src="https://github.com/user-attachments/assets/b55bd3c7-0b9f-4798-910a-70d3d02da0e7" />